.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
```

Comment:
I think we should move this file out of `.github`, into the root of the project, because some people (at least I do), go looking for a `CONTRIBUTING.md`, but wouldn't think to look in `.github` (Github added that feature quite recently).